lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -523,11 +528,10 @@ def run(self, **kwargs):
                         raise ValueError(
                                 f"Validation for data device {device} failed\n"
                                 "Please provide the correct device")
-            cvg_list.insert(self.cvg_count, {'data_devices': data_devices, 'metadata_devices': metadata_devices})
-            self.cvg_count = self.cvg_count + 1
-            self.update_pillar_and_conf('cvg', self.cvg_count)
+            cvg_list.insert(current_cvg_count, {'cvg_name': cvg_name, 'data_devices': data_devices, 'metadata_devices': metadata_devices})
```

Comment:
Is the convention correct here?
`data-devices, metadata-devices`

Reply:
yes, those are variable names.